dts/arm/stm32f411e_disco.dts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017 Fenix Engineering Solutions
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32f411.dtsi>
+
+/ {
+	model = "STMicroelectronics STM32F411E-DISCO board";
+	compatible = "st,stm32f411e-disco", "st,stm32f411";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&usart1 {
+	current-speed = <115200>;
+	pinctrl-0 = <&usart1_pins_a>;
+	pinctrl-names = "default";
+	status = "ok";
+};
```

Comment:
If you choose to activate USART1, update pinmux.c accordingly

Reply:
Sure! USART1 is not needed, so I'll remove it. And pinout is used by USB and I2C (support to be added) so there will be future conflicts if enabled.
Thank you!